.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,48 @@
+<!--
+PLEASE HELP US PROCESS GITHUB ISSUES FASTER BY PROVIDING THE FOLLOWING INFORMATION.
+
+NOTE: IF YOU HAVE A *QUESTION*, PLEASE POST IT ON THE OKTA DEVELOPER FORUM (https://devforum.okta.com) INSTEAD. ISSUES IN THIS REPOSITORY ARE RESERVED FOR BUG REPORTS AND FEATURE REQUESTS.
+-->
+
+## I'm submitting this issue for the package(s):
+- [ ] jwt-verifier
```

Comment:
nit: I'm a bit of a markdown formatting nut, so lets make sure to have newlines before ordered/unordered lists:

```
## I'm submitting this issue for the package(s):

- [ ] jwt-verifier
```